package.json

```diff
@@ -19,9 +21,10 @@
   ],
   "dependencies": {
     "codecov": "^3.1.0",
+    "estraverse-fb": "1.3.2",
+    "flow-mono-cli": "^1.4.3",
```

Comment:
Can you use `flow-bin` ? and can you put it in `devDependencies` please?

Reply:
✅